src/commands/actors/call.ts

```diff
@@ -93,10 +93,6 @@ export class ActorsCallCommand extends ApifyCommand<typeof ActorsCallCommand> {
 			waitForFinish: 2, // NOTE: We need to wait some time to Apify open stream and we can create connection
 		};
 
-		const waitForFinishMillis = Number.isNaN(this.flags.waitForFinish)
```

Comment:
maybe its done elsewhere, but i'd guess you also need to remove the `waitForFinish` flag somewhere (at least on type level)?

Reply:
It was removed from `src/lib/commands/run-on-cloud.ts`